custom_components/smartthinq_sensors/wideq/airpurifier.py

```diff
@@ -112,15 +123,39 @@ def pm10(self):
         key = self._get_state_key(AIR_PURIFIER_STATE_PM10)
         return self._data.get(key)
 
+    def _get_lower_filter_life(self):
+        use_time = self._data.get(self._get_state_key(AIR_PURIFIER_STATE_FILTERMNG_USE_TIME))
+        max_time = self._data.get(self._get_state_key(AIR_PURIFIER_STATE_FILTERMNG_MAX_TIME))
```

Comment:
Looks good, just avoid trying operation if `use_time` is `None` (that probably means filter not exist) and return immediately `None` in case.